pkg/loki/modules.go

```diff
@@ -386,6 +387,32 @@ func (t *Loki) initDistributor() (services.Service, error) {
 	return t.distributor, nil
 }
 
+func (t *Loki) initIngestLimitsRing() (_ services.Service, err error) {
+	if !t.Cfg.IngestLimits.Enabled {
+		return nil, nil
+	}
+
+	reg := prometheus.WrapRegistererWithPrefix(t.Cfg.MetricsNamespace+"_", prometheus.DefaultRegisterer)
```

Comment:
Will we get duplicate metrics registration panics here, as the prefix is the same?

Reply:
:thinking: not sure we have been using the registerer with prefix call multiple times in `modules.go` so I thought it is a no-op here. OTH you are right if we ever call this method twice which is is not the case for distinct targets like `ingest-limits`, however in `all` it is something to investigate

Reply:
Follow up this not even a problem with target `all` because we don't start multiple rings in the same binary there.